xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -39,10 +51,21 @@
         <Parameter Name="state" Type="System.Text.Json.JsonWriterState" />
       </Parameters>
       <Docs>
-        <param name="bufferWriter">To be added.</param>
-        <param name="state">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="bufferWriter">An instance of <see cref="T:System.Buffers.IBufferWriter`1" /> used as a destination for writing JSON text.</param>
+        <param name="state">A struct that contains the reader state. On the first call to the constructor, it should reflect a default state; otherwise, it should capture the state from the previous instance instance of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> and pass that back.</param>
```

Comment:
writer state, not reader state.

Reply:
> On the first call to the constructor, it should reflect a default state

I don't think that captures the intent. The state is being passed in to the ctor. Why is it reflecting anything?

>  it should capture the state from the previous instance instance of the

Not "it", the caller should capture the state...